pages/bookmarks/index.js

```diff
@@ -18,17 +32,20 @@ export default function Bookmarks({ sampleEvents, onToggleFavorite }) {
           {favEvents.length === 0 ? (
             <ErrorMessage message={"🥕 🦫 🥦"} />
           ) : (
-            favEvents.map((favEvent) => (
-              <Fragment key={favEvent.id}>
-                <EventCard
-                  sampleEvent={favEvent}
-                  onToggleFavorite={onToggleFavorite}
-                />
-              </Fragment>
-            ))
+            favEventList
           )}
         </StyledCardSection>
-        <Footer />
+        {isCreating ? (
+          <CreateCard
+            onAddCreateCard={onAddCreateCard}
+            sampleEvents={sampleEvents}
+            onHandleIsCreating={onHandleIsCreating}
+          />
+        ) : null}
+        <Footer
+          sampleEvents={sampleEvents}
+          onHandleIsCreating={onHandleIsCreating}
```

Comment:
ich glaub die sampleEvents musst du hier nicht mitgeben. du nimmst sie auch nicht entgegen in der CreateCard oder Footer Komponente

Reply:
Korrekt! Danke, hab ich ganz übersehen